apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/HHAClaimTransformerV2Test.java

```diff
@@ -135,6 +135,14 @@ public void shouldSetFinalAction() {
   public void shouldSetBillablePeriod() throws Exception {
     // We just want to make sure it is set
     assertNotNull(eob.getBillablePeriod());
+    Extension extension =
+        eob.getBillablePeriod()
+            .getExtensionByUrl("https://bluebutton.cms.gov/resources/variables/claim_query_cd");
+    Coding valueCoding = (Coding) extension.getValue();
+    assertEquals("Final bill", valueCoding.getDisplay());
+    assertEquals("3", valueCoding.getCode());
+    assertEquals(
+        "https://bluebutton.cms.gov/resources/variables/claim_query_cd", valueCoding.getSystem());
```

Comment:
If we're saying claim query is optional, we also need a test for when it is not present to ensure it is not set (negative test) for each of the claims it can appear in. (Probably we want to validate it is NOT passable as null in the claims we expect it in, and able to be not present in the claims we expect it to be absent in)

Reply:
so for all claim types that are not HHA and hospice it is mandatory. This is validated when the data comes in from CCW bc it is required.

Reply:
I wouldn't trust our upstream on this; if we know it should be required, we should be checking it IMO. Trust nothing external!

Reply:
DB constraint..